src/core/CLucene/index/FieldInfos.cpp

```diff
@@ -257,6 +274,9 @@ void FieldInfos::write(IndexOutput* output) const{
 		if (fi->getIndexVersion() > IndexVersion::kV0) {
 			output->writeVInt(static_cast<int32_t>(fi->getIndexVersion()));
 		}
+		if (fi->getIndexVersion() >= IndexVersion::kV3) {
```

Comment:
add test for index version compatibility.

Reply:
Included in testIndexCompressV3.